### ISSUE TITLE
training stops as soon as it iterates!!!

### ISSUE THREAD

**Issue by mutantmassXXX** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#125]
[[Node: batch/padding_fifo_queue_enqueue = QueueEnqueueV2[Tcomponents=[DT_STRING, DT_INT32, DT_FLOAT, DT_INT32, DT_FLOAT, ..., DT_INT32, DT_INT32, DT_INT32, DT_STRING, DT_INT32], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/device:CPU:0"](batch/padding_fifo_queue, IteratorGetNext, Shape_4, IteratorGetNext:1, Shape_3, IteratorGetNext:2, Shape, IteratorGetNext:3, Shape_5, IteratorGetNext:4, Shape_7, IteratorGetNext:5, Shape_1, IteratorGetNext:6, Shape_2, IteratorGetNext:7, Shape_6, IteratorGetNext:8, Shape_10, ToFloat, Shape_8, IteratorGetNext:10, Shape_4, IteratorGetNext:11, Shape_4, IteratorGetNext:12, Shape_4)]]
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:Recording summary at step 0.
INFO:tensorflow:global step 1: loss = 13.6541 (19.544 sec/step)
INFO:tensorflow:global step 1: loss = 13.6541 (19.544 sec/step)
INFO:tensorflow:Finished training! Saving model to disk.
INFO:tensorflow:Finished training! Saving model to disk.

**Comment by mjasonong** [comment]
same problem here. did you already solved it? pls help me :(
